ext/ldap/ldap.c

```diff
@@ -980,7 +980,8 @@ PHP_FUNCTION(ldap_connect)
 
 #ifdef HAVE_ORALDAP
 	if (ZEND_NUM_ARGS() == 3 || ZEND_NUM_ARGS() == 4) {
-		WRONG_PARAM_COUNT;
+		zend_wrong_param_count();
+		RETURN_THROWS();
 	}
```

Comment:
I guess this doesn't really matter as this will soon be removed anyway so using the bad error message provided by `zend_wrong_param_count` is fine